corehq/apps/hqwebapp/views.py

```diff
@@ -93,23 +93,27 @@ def server_error(request, template_name='500.html'):
     traceback_key = uuid.uuid4().hex
     cache.cache.set(traceback_key, traceback_text, 60*60)
 
-    return HttpResponseServerError(t.render(RequestContext(request,
-        {'MEDIA_URL': settings.MEDIA_URL,
-         'STATIC_URL': settings.STATIC_URL,
-         'domain': domain,
-         '500traceback': traceback_key,
-        })))
+    return HttpResponseServerError(t.render(
+        RequestContext(request, {
+            'MEDIA_URL': settings.MEDIA_URL,
+            'STATIC_URL': settings.STATIC_URL,
+            'domain': domain,
+            '500traceback': traceback_key,
+        }).dicts[0]
+    ))
```

Comment:
I think the preferable way to do this would be to no longer bother assembling a `RequestContext` object - `render` takes an optional `request` argument which I believe does the same thing
https://docs.djangoproject.com/en/1.10/topics/templates/#django.template.backends.base.Template.render

```python
return HttpResponseServerError(t.render(
    {
        'MEDIA_URL': settings.MEDIA_URL,
        'STATIC_URL': settings.STATIC_URL,
        'domain': domain,
        '500traceback': traceback_key,
    },
    request
))
```